package.json

```diff
@@ -3561,6 +3561,7 @@
         "webpack-cli": "^4.9.1",
         "webpack-dev-server": "^4.13.3"
     },
+    "sdk-codegen-version": "3.345.0",
```

Comment:
is this a Toolkit-owned custom property? maybe we should put all our properties in a `awstoolkit: {}` object